src/metatrain/experimental/soap_bpnn/trainer.py

```diff
@@ -290,7 +290,18 @@ def train(
                 targets = average_by_num_atoms(targets, systems, per_structure_targets)
 
                 train_loss_batch = loss_fn(predictions, targets)
-                train_loss_batch.backward()
+
+                with warnings.catch_warnings():
+                    warnings.filterwarnings(
+                        action="ignore",
+                        message="second derivatives with respect to positions",
+                    )
+                    warnings.filterwarnings(
+                        action="ignore",
+                        message="second derivatives with respect to cell matrix",
+                    )
+                    train_loss_batch.backward()
```

Comment:
I think we should have this at the top of the file to try to keep the training code a bit more concise

Reply:
this will ignore this warning at all time, even in other architectures. If you are fine with this I can move them!

Reply:
I could not move them to the top of the file for some reason, so I moved them to the beginning of the function instead!